model/articles-model.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../db/connection");
+
+function getArticleById(id) {
+  let sqlString = `SELECT * FROM articles`;
+    if (id) {
+      sqlString += ` WHERE article_id=$1;`;
```

Comment:
You will need the 'Where' clause every time you do a query in this function so you can have it in one statement and handle the situation when ID is not coming in.